modules/cluster-management/src/main/java/org/apache/ignite/internal/cluster/management/ClusterManagementGroupManager.java

```diff
@@ -366,6 +381,32 @@ private void onElectedAsLeader(long term) {
                         LOG.warn("Error when executing onLeaderElected callback", e);
                     }
                 });
+
+        raftServiceAfterJoin().thenComposeAsync(service -> {
```

Comment:
Check the current ClusterStte and apply configuration changes if restAuthToApply is not null. 
No, we are not going to apply changes again, because restAuthToApply will be null the next time.